components/CollectiveNavbar.js

```diff
@@ -305,6 +306,13 @@ export const getFilteredSectionsForCollective = (collective, isAdmin, isHostAdmi
     toRemove.add(Sections.RECURRING_CONTRIBUTIONS);
   }
 
+  // don't display for TYPE=COLLECTIVE if no active contributions
+  if (collective.type === CollectiveType.COLLECTIVE) {
+    if (!collective.ordersFromCollective.filter(collective => collective.isSubscriptionActive === true).length) {
```

Comment:
There's currently a bug that prevents it, but this function is normally also called for other pages that have the collective navbar (will be fixed by https://github.com/opencollective/opencollective-frontend/pull/4463). Most of  these pages don't fetch the data for `ordersFromCollective`, so we need to be careful here. It may also be easier and more performant to use `some`:

```suggestion
    if (!collective.ordersFromCollective?.some(collective => collective.isSubscriptionActive)) {
```

FYI, the intent of https://github.com/opencollective/opencollective-api/pull/3992 is to prevent these data inconsistencies between pages to make sure we'll display the same navbar everywhere.